saritasa_invocations/cruft.py

```diff
@@ -53,8 +54,8 @@ def create_project(
     config = _config.Config.from_context(context)
     tmp_folder = config.cruft.project_tmp_folder
     printing.print_success(f"Recreating tmp ({tmp_folder}) folder")
-    context.run(f"rm -rf {tmp_folder}")
-    context.run(f"mkdir -p {tmp_folder}")
+    shutil.rmtree(tmp_folder, ignore_errors=True)
```

Comment:
Let's use https://docs.python.org/3/library/pathlib.html#pathlib.Path.rmdir here

And general idea - use `pathlib` whenever possible.
I guess there is no need to use `shutil` for copying, because actually these tasks creates new files based on some template

Reply:
`Remove this directory. The directory must be empty.`

Reply:
> And general idea - use pathlib whenever possible.

For that we have ruff)